spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierApplications.java

```diff
@@ -986,11 +986,6 @@ private static TypeQualifierAnnotation computeEffectiveTypeQualifierAnnotation(T
             // default annotations
         }
 
-        /** private methods don't inherit from class or package scope */
-        if (xmethod.isPrivate()) {
-            stopAtMethodScope = true;
-        }
-
```

Comment:
I'm still not sure about the intention behind this code. 
But commit message of b1fcbae0c11852eb42e5e2f7e1eca61003aa4a7b tells nothing, so I agree to change the behavior to make it intuitive.